### DISCUSSION TITLE
Is there any breaking changes in v7?

### DISCUSSION THREAD

**Discussion by Satish7897** [reduxjs/react-redux#1853] [Q&A]
`v7`

**Comment by markerikson** [comment]
This is a very unclear question :)  Can you clarify what you're asking about, specifically?  And also, do you mean v7, or v8?  The current version is 7.2.6.  v8 is currently in beta.

FWIW both of them do have "breaking" changes - that's why they're major version bumps.  But, the "breaking" bit in both cases is very small and unlikely to affect most codebases.
